src/utils/googleTagManager.ts

```diff
@@ -87,7 +87,10 @@ export const unloadGoogleTagManager = (): void => {
     { name: '_gat', path: '/' },
     { name: '_gid', path: '/' },
   ]
-  removeCookies(GOOGLE_ANALYTICS_COOKIE_LIST)
+
+  if (window.dataLayer) {
```

Comment:
Why not an early return in the beginning of this fn?

Reply:
Are you suggesting before the constant? A guard clause for just `removeCookies` doesn't seem worthwhile.